lib/ukdebug/gdbstub.c

```diff
@@ -94,6 +94,28 @@ UK_LIBPARAM_PARAM_ALIAS(gdb_cons, &gdb_backing_console_id, __u16,
 			"gdb backing console ID");
 #endif /* CONFIG_LIBUKLIBPARAM */
 
+/* This is a tristate value controlling the behavior of the gdb virtual
+ * console. It's value is interpreted as follows:
+ * - (> 0): always register the virtual console
```

Comment:
It has been addressed. On my github it shows up like this:

```
/* This is a tristate value controlling the behavior of the gdb stub's virtual
 * console. The virtual console implements a console device for `ukconsole`.
 * This console device is called "virtual" because it's not a driver for a
 * real console device. The virtual console sends all console output to the
 * gdb host using the 'O' packet. The backing console is used to transmit this
 * packet like any other. With the virtual console, the backing console can be
 * used for kernel console output and for gdb communication at the same time.
 *
 * The value of this tristate is interpreted as follows:
 * - (> 0): always register the virtual console
[...]
```